frontend/src/components/search/SearchBar.tsx

```diff
@@ -16,27 +17,35 @@ export default function SearchBar({ initialQueryParams }: SearchBarProps) {
   };
 
   return (
-    <div className="usa-search usa-search--big" role="search">
-      <label className="usa-sr-only" htmlFor="query">
-        Search
+    <div className="margin-top-5 margin-bottom-2">
+      <label
+        htmlFor="query"
+        className="font-sans-lg display-block margin-bottom-2"
+      >
+        <span className="text-bold">Search terms </span>
+        <small className="display-inline-block">
+          Enter keywords, opportunity numbers, or assistance listing numbers
+        </small>
```

Comment:
Moving the `label` outside the `.usa-search` to avoid the flex grid styles it gets that force it left of the `input`